action/completion_generate.go

```diff
@@ -22,17 +22,19 @@ var CompletionGenerate = &cli.Command{
 
 		// Shell Flags
 
-		&cli.BoolFlag{
+		&cli.StringFlag{
 			EnvVars: []string{"VELA_BASH", "COMPLETION_BASH"},
 			Name:    "bash",
 			Aliases: []string{"b"},
 			Usage:   "generate a bash auto completion script",
+			Value:   "false",
 		},
-		&cli.BoolFlag{
+		&cli.StringFlag{
 			EnvVars: []string{"VELA_ZSH", "COMPLETION_ZSH"},
 			Name:    "zsh",
 			Aliases: []string{"z"},
 			Usage:   "generate a zsh auto completion script",
+			Value:   "false",
 		},
 	},
 	CustomHelpTemplate: fmt.Sprintf(`%s
```

Comment:
One last question with this change.. Does it still work to pass the flag alone this way without the explicit value?

In case it's unclear, previously before this change, you could pass the bool flag alone and it was set it to `true`.

i.e.

* `vela generate completion --bash`
* `vela generate completion --zsh`

With this new change, would I be required to pass a `true` or `=true` after the flag?

i.e.

* `vela generate completion --bash true`
* `vela generate completion --bash=true`
* `vela generate completion --zsh true`
* `vela generate completion --zsh=true`

Reply:
> With this new change, would I be required to pass a true or =true after the flag?

Yes - the updated arguments will default to `false` if you utilize the flag without a value.

Reply:
Ok 👍 

The reason why I ask is because I think we'll need to update our `Examples` section in the help output in this PR for the commands that had this change.

To ensure I'm clear, all our CLI commands and subcommands respect a `--help` flag you can pass.

i.e. `vela generate completion --help`

If we merge this in its current state, then our examples would be wrong in that output:

https://github.com/go-vela/cli/blob/652a53ba76ee2053dae1d27b6a34d8aab643a347/action/completion_generate.go#L42-L45

I believe it would have to show something like:

```
   1. Enable auto completion for the current bash session. 
     $ source <({{.HelpName}} --bash true) 
```

Reply:
👍 I'll go ahead and update the examples sometime this afternoon.

Reply:
Changes have been made.